config.yml

```diff
@@ -5,13 +5,21 @@ defaults:
   # TaskCluster credentials
   taskcluster:
     rootUrl: !env TASKCLUSTER_ROOT_URL
+    queue: none
     credentials:
       clientId: !env TASKCLUSTER_CLIENT_ID
       accessToken: !env TASKCLUSTER_ACCESS_TOKEN
 
   monitoring:
     enable: !env:bool MONITORING_ENABLE
 
+  worker:
+    workerId: "succeed"
+    workerType: "succeed"
```

Comment:
No need to set these in the configuraiton -- they will be different for each TaskQueue instance.